guide/onboarding/manual-backup.md

```diff
@@ -0,0 +1,287 @@
+---
+layout: guide
+title: Manual backup
+description: How to backup a recovery phrase manually
+grand_parent: Onboarding
+parent: Backing up a recovery phrase
+nav_order: 2
+permalink: /guide/onboarding/backing-up-a-recovery-phrase/manual-backup/
+main_classes: -no-top-padding
+---
+
+# Manual backup
+
+### How it Works
+
+As previously stated, onboarding flows that require users to manually back up their seed phrase typically shows users their respective 12-24 word phrase, expecting them to then write it down on a piece of paper and store it in a safe (but memorable location). In the case that a user’s device breaks or  is stolen, the user can recover their funds and wallet by correctly entering their seed phrase. The [private key management]({{ '/guide/private-key-management/introduction' | relative_url }}) section dives further into the technical details of this scheme if interested in learning more.
```

Comment:
`As previously stated` made more sense when this was all one big page, but now it's not so clear. I'd either remove it or turn it into a link to where it was stated before.